testapp/settings.py

```diff
@@ -24,8 +24,7 @@ def get_db():
 
 DATABASES = {"default": get_db()}
 
-# TODO: update to BigAutoField and makemigrations once we drop 2.2 support
-DEFAULT_AUTO_FIELD = "django.db.models.AutoField"
+DEFAULT_AUTO_FIELD = "django.db.models.BigAutoField"
```

Comment:
this actually needs to be set in the  `AppConfig` - the models themselves when installed as part of OZ don't use this setting so you'll end up with the reverse migration again.

Reply:
yeah, my bad!